cmd/admin-policy-info.go

```diff
@@ -18,25 +18,34 @@
 package cmd
 
 import (
+	"os"
+
 	"github.com/fatih/color"
 	"github.com/minio/cli"
 	"github.com/minio/madmin-go"
 	"github.com/minio/mc/pkg/probe"
 	"github.com/minio/pkg/console"
 )
 
+var policyInfoFlags = []cli.Flag{
+	cli.StringFlag{
+		Name:  "policy-file, f",
+		Usage: "additionally (over-)write policy JSON to given file",
```

Comment:
I think maybe let's avoid printing in the stdout when this flag is passed

Reply:
@donatello ^^